build_runner/lib/src/entrypoint/options.dart

```diff
@@ -283,30 +271,32 @@ Map<String, Map<String, dynamic>> _parseBuilderConfigOverrides(
   return builderConfigOverrides;
 }
 
-/// Returns a map of output directory to root input directory to be used
-/// for merging.
+/// Returns a list of BuiltTargets with output information.
```

Comment:
BuiltTargets -> BuildDirectory.

[nit] Remove the "list of" from the wording.

`/// Returns the BuildDirectories requested for output arguments.`